source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && !defined(PF_DISTRO_ULINUX)
```

Comment:
Are you saying that, without a universal build, this routine is never called, ever?